packages/apps/tools/src/pages/transactions/module-explorer/utils.ts

```diff
@@ -25,8 +25,8 @@ export const getQueryValue = (
 export const getCookieValue = (
   needle: string,
   haystack: NextApiRequestCookies,
-  defaultValue?: string,
-): string | null => {
+  defaultValue?: string | [any],
```

Comment:
Why does the `defaultValue` need to be changed?

Reply:
We return the array of networks here, so if they're not in the cookie, the default will be an empty array

Reply:
But the `defaultValue` is not being given as a parameter for the networks cookie retrieval, right?

Reply:
Ah, yeah... I removed the default value in the params. I will remove it here as well